extra_tests/snippets/isinstance.py

```diff
@@ -53,4 +60,4 @@ class ReturnInt(metaclass=MCReturnInt):
 
 assert isinstance("a", ReturnInt) is True
 
-assert isinstance(1, ((int, float,), str))
+assert isinstance(1, ((int, float,), str))
```

Comment:
revert this line not to make whitespace change

Reply:
would you also clear this change not to be made?

Reply:
applied it ! Sorry for make you cumbersome 😂